apps/etterlatte-vedtaksvurdering/src/main/kotlin/vedtaksvurdering/VedtaksvurderingService.kt

```diff
@@ -178,8 +194,13 @@ class VedtaksvurderingService(
         verifiserGyldigVedtakStatus(vedtak.status, listOf(VedtakStatus.FATTET_VEDTAK))
         attestantHarAnnenIdentEnnSaksbehandler(vedtak.vedtakFattet!!.ansvarligSaksbehandler, brukerTokenInfo)
 
-        val (behandling, _, _, sak) = hentDataForVedtak(behandlingId, brukerTokenInfo)
+        val (behandling, vilkaarsvurdering, beregningOgAvkorting, sak) =
+            hentDataForVedtak(behandlingId, brukerTokenInfo)
+
         verifiserGyldigVedtakForRevurdering(behandling, vedtak)
+        verifiserGrunnlagVersjon(vilkaarsvurdering, beregningOgAvkorting)
+        // TODO: Låse versjon i grunnlag
```

Comment:
Jeg tenker at enten verifiserGrunnlagVersjon eller låsning av grunnlag bør få med sjekken om nyeste hendelse i behandlingen matcher grunnlaget som er brukt i beregning / vilkårsvurdering også. Kanskje låsing av grunnlag kan ta imot en versjon, og feile hvis man prøver å låse til en versjon annen enn det behandlingen har nå f.eks.

Reply:
Godt innspill! Slo meg også at kasting av feil ikke skulle være med riktig enda, så har fjernet det i siste commit. Kommer POC for versjonshåndtering i neste PR.